packages/wallet/test/e2e/config.ts

```diff
@@ -68,7 +71,8 @@ const blockfrostApi = [
   env.WALLET_PASSWORD,
   env.TX_SUBMIT_PROVIDER,
   env.ASSET_PROVIDER,
-  env.NETWORK_INFO_PROVIDER
+  env.NETWORK_INFO_PROVIDER,
+  env.CHAIN_HISTORY_PROVIDER
```

Comment:
Isn't this missing `env.UTXO_PROVIDER`?

And shouldn't it be `env.WALLET_PROVIDER` instead of `env.WALLET_PASSWORD`?

The same is happening here https://github.com/input-output-hk/cardano-js-sdk/pull/267/files#diff-b5bf86cc0c1c2281ead2068f3647944bb80c64c141fe77e6a2aebff890cb84b4R58

Reply:
> And shouldn't it be env.WALLET_PROVIDER instead of env.WALLET_PASSWORD

Both are used. WALLET_PASSWORD is used for creating `InMemoryKeyAgent`

Reply:
`WALLET_PASSWORD` is being used to conditionally instantiate `BlockFrostAPI`: 

https://github.com/input-output-hk/cardano-js-sdk/blob/3a202ea407b9260b8598a3e1097d535cb3ca7f34/packages/wallet/test/e2e/config.ts#L69-L84

It'd be nice to isolate that as a named function, to better describe it :sunglasses:

Reply:
Oh this doesn't look right, yeah should be `WALLET_PROVIDER` in here